### PR TITLE
Update highlight_regexes

### PR DESCRIPTION
**Pull request by janisz**
Only test failures and real panics should be highlighted not any occurence of word FAIL or panic as it can be a part of test name.

Example Job with false postives https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/2726/pull-ci-stackrox-stackrox-master-go-unit-tests/1560204625850667008

<details>
<pre>
2683 skipped lines...
PASS
coverage: 84.1% of statements
ok  	github.com/stackrox/rox/central/compliance/checks/nist800-190/check411	0.498s	coverage: 84.1% of statements
=== RUN   TestNIST412_Success
--- PASS: TestNIST412_Success (0.00s)
=== RUN   TestNIST412_FAIL
--- PASS: TestNIST412_FAIL (0.00s)
PASS
coverage: 91.2% of statements
ok  	github.com/stackrox/rox/central/compliance/checks/nist800-190/check412	0.413s	coverage: 91.2% of statements
=== RUN   TestNIST414_Success
--- PASS: TestNIST414_Success (0.00s)
=== RUN   TestNIST414_FAIL
--- PASS: TestNIST414_FAIL (0.00s)
PASS
coverage: 100.0% of statements
ok  	github.com/stackrox/rox/central/compliance/checks/nist800-190/check414	0.420s	coverage: 100.0% of statements
=== RUN   TestNIST422_Success
--- PASS: TestNIST422_Success (0.00s)
3939 skipped lines...
networkgraph/service: 2022/08/18 10:21:41.309123 service_impl.go:274: Warn: Number of deployments is too high to be rendered in Network Graph: 2001
=== RUN   TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit/Specific_Env_value
=== CONT  TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit
    env_isolator.go:41: EnvIsolator: Setting ROX_MAX_DEPLOYMENTS_NETWORK_GRAPH to 1000
networkgraph/service: 2022/08/18 10:21:41.310070 service_impl.go:274: Warn: Number of deployments is too high to be rendered in Network Graph: 1001
=== RUN   TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit/Incorrect_env_value_shouldn't_panic
=== CONT  TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit
    env_isolator.go:41: EnvIsolator: Setting ROX_MAX_DEPLOYMENTS_NETWORK_GRAPH to dummy
networkgraph/service: 2022/08/18 10:21:41.311092 service_impl.go:274: Warn: Number of deployments is too high to be rendered in Network Graph: 2001
--- PASS: TestNetworkGraph (0.04s)
    --- PASS: TestNetworkGraph/TestCreateExternalNetworkEntity (0.00s)
5 skipped lines...
    --- PASS: TestNetworkGraph/TestNetworkGraphConfiguration (0.00s)
    --- PASS: TestNetworkGraph/TestPatchExternalNetworkEntity (0.00s)
    --- PASS: TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit (0.00s)
        --- PASS: TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit/Default (0.00s)
        --- PASS: TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit/Specific_Env_value (0.00s)
        --- PASS: TestNetworkGraph/TestReturnErrorIfNumberOfNodesExceedsLimit/Incorrect_env_value_shouldn't_panic (0.00s)
PASS
coverage: 76.0% of statements
ok  	github.com/stackrox/rox/central/networkgraph/service	1.108s	coverage: 76.0% of statements
=== RUN   TestNetPolDataStore
=== PAUSE TestNetPolDataStore
3626 skipped lines...
coverage: 4.0% of statements
ok  	github.com/stackrox/rox/central/role/service	1.023s	coverage: 4.0% of statements
?   	github.com/stackrox/rox/central/role/store/permissionset/postgres	[no test files]
?   	github.com/stackrox/rox/central/role/store/role/postgres	[no test files]
?   	github.com/stackrox/rox/central/role/store/simpleaccessscope/postgres	[no test files]
FAIL	github.com/stackrox/rox/central/sac/authorizer [build failed]
=== RUN   TestHandler
=== RUN   TestHandler/TestGenerateScannerHTTPHandler
=== CONT  TestHandler
    env_isolator.go:41: EnvIsolator: Setting ROX_MTLS_CA_FILE to /go/src/github.com/stackrox/stackrox/pkg/mtls/testutils/testdata/central-certs/ca.pem
    env_isolator.go:41: EnvIsolator: Setting ROX_MTLS_CA_KEY_FILE to /go/src/github.com/stackrox/stackrox/pkg/mtls/testutils/testdata/central-certs/ca-key.pem
1348 skipped lines...
pkg/rocksdb: 2022/08/18 10:24:28.625254 rocksdb.go:70: Info: Closed RocksDB
=== RUN   TestViolations/TestWithDeploymentImage
pkg/rocksdb: 2022/08/18 10:24:28.662666 rocksdb.go:51: Info: Signaling RocksDB to close
pkg/rocksdb: 2022/08/18 10:24:28.662805 rocksdb.go:64: Info: Closing RocksDB now that all operations have completed
pkg/rocksdb: 2022/08/18 10:24:28.[6646](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/2726/pull-ci-stackrox-stackrox-master-go-unit-tests/1560204625850667008#1:build-log.txt%3A6646)33 rocksdb.go:70: Info: Closed RocksDB
--- FAIL: TestViolations (2.85s)
    --- PASS: TestViolations/TestAlertWithoutPolicy (0.07s)
    --- PASS: TestViolations/TestAlertWithoutViolations (0.04s)
    --- PASS: TestViolations/TestCheckpointFromAlertIDFiltering (0.05s)
    --- PASS: TestViolations/TestCheckpointInTheFuture (0.04s)
    --- PASS: TestViolations/TestCheckpointIteration (0.19s)
64 skipped lines...
        --- PASS: TestViolations/TestResponsePagination/{1_7} (0.05s)
        --- PASS: TestViolations/TestResponsePagination/{1_10} (0.04s)
        --- PASS: TestViolations/TestResponsePagination/{2_5} (0.03s)
        --- PASS: TestViolations/TestResponsePagination/{10_1} (0.04s)
    --- PASS: TestViolations/TestViolationIdsAreDistinct (0.06s)
    --- FAIL: TestViolations/TestViolationsHandlerError (0.03s)
    --- PASS: TestViolations/TestViolationsHandlerWriteError (0.04s)
    --- PASS: TestViolations/TestWithDeploymentImage (0.04s)
FAIL
coverage: 62.2% of statements
FAIL	github.com/stackrox/rox/central/splunk	3.788s
=== RUN   TestSummaryTypeToResourceMap
--- PASS: TestSummaryTypeToResourceMap (0.00s)
PASS
coverage: 0.0% of statements
ok  	github.com/stackrox/rox/central/summary/service	0.951s	coverage: 0.0% of statements
10761 skipped lines...
    --- PASS: TestErrorToGrpcCodeStreamInterceptor/Error_is_already_a_gRPC_status_error_(w/_status_code)_->_don't_modify,_just_pass_through (0.00s)
    --- PASS: TestErrorToGrpcCodeStreamInterceptor/Error_is_one_of_types_from_pkg/errorhelpers_(ErrNotFound_etc.)_->_map_to_correct_gRPC_code,_preserve_error_message (0.00s)
    --- PASS: TestErrorToGrpcCodeStreamInterceptor/Error_is_not_a_gRPC_status_error_and_not_a_known_error_type_->_set_error_to_internal (0.00s)
=== RUN   TestPanicOnInvariantViolationUnaryInterceptor
=== RUN   TestPanicOnInvariantViolationUnaryInterceptor/Error_is_nil_->_do_nothing,_just_pass_through
=== RUN   TestPanicOnInvariantViolationUnaryInterceptor/Error_is_ErrInvariantViolation_->_panic
=== RUN   TestPanicOnInvariantViolationUnaryInterceptor/Error_is_not_ErrInvariantViolation_->_do_nothing,_just_pass_through
--- PASS: TestPanicOnInvariantViolationUnaryInterceptor (0.00s)
    --- PASS: TestPanicOnInvariantViolationUnaryInterceptor/Error_is_nil_->_do_nothing,_just_pass_through (0.00s)
    --- PASS: TestPanicOnInvariantViolationUnaryInterceptor/Error_is_ErrInvariantViolation_->_panic (0.00s)
    --- PASS: TestPanicOnInvariantViolationUnaryInterceptor/Error_is_not_ErrInvariantViolation_->_do_nothing,_just_pass_through (0.00s)
=== RUN   TestPanicOnInvariantViolationStreamInterceptor
=== RUN   TestPanicOnInvariantViolationStreamInterceptor/Error_is_nil_->_do_nothing,_just_pass_through
=== RUN   TestPanicOnInvariantViolationStreamInterceptor/Error_is_ErrInvariantViolation_->_panic
=== RUN   TestPanicOnInvariantViolationStreamInterceptor/Error_is_not_ErrInvariantViolation_->_do_nothing,_just_pass_through
--- PASS: TestPanicOnInvariantViolationStreamInterceptor (0.00s)
    --- PASS: TestPanicOnInvariantViolationStreamInterceptor/Error_is_nil_->_do_nothing,_just_pass_through (0.00s)
    --- PASS: TestPanicOnInvariantViolationStreamInterceptor/Error_is_ErrInvariantViolation_->_panic (0.00s)
    --- PASS: TestPanicOnInvariantViolationStreamInterceptor/Error_is_not_ErrInvariantViolation_->_do_nothing,_just_pass_through (0.00s)
=== RUN   TestErrToGRPCStatus
=== RUN   TestErrToGRPCStatus/Wrapped_Sentinel
=== RUN   TestErrToGRPCStatus/NotFound
=== RUN   TestErrToGRPCStatus/Wrapped
2012 skipped lines...
=== CONT  TestImageFlavor/TestGetImageFlavorFromEnv
    env_isolator.go:41: EnvIsolator: Setting ROX_IMAGE_FLAVOR to opensource
=== RUN   TestImageFlavor/TestGetImageFlavorFromEnv/#00
=== CONT  TestImageFlavor/TestGetImageFlavorFromEnv
    env_isolator.go:41: EnvIsolator: Setting ROX_IMAGE_FLAVOR to 
pkg/images/defaults: 2022/08/18 10:27:58.658950 flavor.go:289: Warn: Environment variable ROX_IMAGE_FLAVOR not set, this will cause a panic in release build. Assuming this code is executed in unit test session and using 'development_build' as default.
=== RUN   TestImageFlavor/TestOpenSourceImageFlavorDevReleaseTags
--- PASS: TestImageFlavor (0.05s)
    --- PASS: TestImageFlavor/TestChartRepoAndIcon (0.00s)
        --- PASS: TestImageFlavor/TestChartRepoAndIcon/development_build (0.00s)
        --- PASS: TestImageFlavor/TestChartRepoAndIcon/rhacs (0.00s)
1534 skipped lines...
ok  	github.com/stackrox/rox/pkg/mathutil	0.064s	coverage: 8.6% of statements
=== RUN   TestNewDefaultHTTPServer_default_port
--- PASS: TestNewDefaultHTTPServer_default_port (0.00s)
=== RUN   TestNewDefaultHTTPServer_with_port
--- PASS: TestNewDefaultHTTPServer_with_port (0.00s)
=== RUN   TestNewDefaultHTTPServer_dev_panic
--- PASS: TestNewDefaultHTTPServer_dev_panic (0.00s)
=== RUN   TestNewDefaultHTTPServer_release_nil
--- SKIP: TestNewDefaultHTTPServer_release_nil (0.00s)
PASS
coverage: 7.6% of statements
ok  	github.com/stackrox/rox/pkg/metrics	0.362s	coverage: 7.6% of statements
2118 skipped lines...
    --- PASS: TestSink/TestPublishAuthzTrace (0.00s)
    --- PASS: TestSink/TestSubscribe (0.00s)
PASS
coverage: 16.3% of statements
ok  	github.com/stackrox/rox/pkg/sac/observe	0.452s	coverage: 16.3% of statements
FAIL	github.com/stackrox/rox/pkg/sac/tests [build failed]
=== RUN   TestRunE
--- PASS: TestRunE (0.00s)
PASS
coverage: 75.0% of statements
ok  	github.com/stackrox/rox/pkg/safe	0.026s	coverage: 75.0% of statements
755 skipped lines...
=== RUN   TestMap/simple_func_on_string
=== RUN   TestMap/same_func,_but_with_ptr
=== RUN   TestMap/func_of_int_to_string_array
=== RUN   TestMap/extract_element_from_struct
=== RUN   TestMap/extract_element_from_struct_with_pointer
=== RUN   TestMap/extract_element_from_struct_with_pointer_to_pointer,_should_panic
=== RUN   TestMap/extract_element_from_struct_with_pointer_to_pointer
--- PASS: TestMap (0.00s)
    --- PASS: TestMap/simple_func_on_string (0.00s)
    --- PASS: TestMap/same_func,_but_with_ptr (0.00s)
    --- PASS: TestMap/func_of_int_to_string_array (0.00s)
    --- PASS: TestMap/extract_element_from_struct (0.00s)
    --- PASS: TestMap/extract_element_from_struct_with_pointer (0.00s)
    --- PASS: TestMap/extract_element_from_struct_with_pointer_to_pointer,_should_panic (0.00s)
    --- PASS: TestMap/extract_element_from_struct_with_pointer_to_pointer (0.00s)
=== RUN   TestReverseInPlace
--- PASS: TestReverseInPlace (0.00s)
=== RUN   TestReversed
--- PASS: TestReversed (0.00s)
863 skipped lines...
ok  	github.com/stackrox/rox/roxctl/common/environment	0.593s	coverage: 28.6% of statements
=== RUN   TestLogger
=== RUN   TestLogger/(TOTAL:_0,_\x1b[34;2mLOW\x1b[0m:_0,_\x1b[33mMEDIUM\x1b[0m:_0,_\x1b[95mHIGH\x1b[0m:_0,_\x1b[31;1mCRITICAL\x1b[0m:_0)_
=== RUN   TestLogger/\x1b[94mINFO:_(TOTAL:_0,_LOW:_0,_MEDIUM:_0,_HIGH:_0,_CRITICAL:_0)_\x1b[0m
=== RUN   TestLogger/\x1b[95mWARN:_(TOTAL:_0,_LOW:_0,_MEDIUM:_0,_HIGH:_0,_CRITICAL:_0)_\x1b[0m
=== RUN   TestLogger/\x1b[31;1mERROR:_(TOTAL:_0,_LOW:_0,_MEDIUM:_0,_HIGH:_0,_CRITICAL:_0)_\x1b[0m
--- PASS: TestLogger (0.00s)
    --- PASS: TestLogger/(TOTAL:_0,_\x1b[34;2mLOW\x1b[0m:_0,_\x1b[33mMEDIUM\x1b[0m:_0,_\x1b[95mHIGH\x1b[0m:_0,_\x1b[31;1mCRITICAL\x1b[0m:_0)_ (0.00s)
    --- PASS: TestLogger/\x1b[94mINFO:_(TOTAL:_0,_LOW:_0,_MEDIUM:_0,_HIGH:_0,_CRITICAL:_0)_\x1b[0m (0.00s)
    --- PASS: TestLogger/\x1b[95mWARN:_(TOTAL:_0,_LOW:_0,_MEDIUM:_0,_HIGH:_0,_CRITICAL:_0)_\x1b[0m (0.00s)
    --- PASS: TestLogger/\x1b[31;1mERROR:_(TOTAL:_0,_LOW:_0,_MEDIUM:_0,_HIGH:_0,_CRITICAL:_0)_\x1b[0m (0.00s)
PASS
coverage: 100.0% of statements
ok  	github.com/stackrox/rox/roxctl/common/logger	0.040s	coverage: 100.0% of statements
=== RUN   TestColorPrinter
--- PASS: TestColorPrinter (0.00s)
2163 skipped lines...
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent1	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent2	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent3	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent4	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testshortcircuit	[no test files]
FAIL
make: *** [Makefile:464: go-unit-tests] Error 2
INFO: Thu Aug 18 10:34:23 UTC 2022: Saving junit XML report
+ /go/bin/go-junit-report
go: downloading github.com/jstemmer/go-junit-report/v2 v2.0.0
/go/src/github.com/stackrox/stackrox/scripts/generate-junit-reports.sh
INFO: Thu Aug 18 10:34:25 UTC 2022: Copying test results from junit-reports to reports
Unit tests failed
INFO: Thu Aug 18 10:34:25 UTC 2022: Executing a general purpose exit trap for CI
Exit code is: 1
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-08-18T10:34:26Z"}
error: failed to execute wrapped command: exit status 1 
INFO[2022-08-18T10:34:34Z] Step go-unit-tests-stackrox-initial failed after 23m50s. 
INFO[2022-08-18T10:34:34Z] Step phase test failed after 23m50s.         
INFO[2022-08-18T10:34:34Z] Running multi-stage phase post               
INFO[2022-08-18T10:34:34Z] Step phase post succeeded after 0s.          
INFO[2022-08-18T10:34:36Z] Ran for 35m19s                               
ERRO[2022-08-18T10:34:36Z] Some steps failed:                           
ERRO[2022-08-18T10:34:36Z] 
10 skipped lines...
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent1	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent2	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent3	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testparent4	[no test files]
?   	github.com/stackrox/rox/tools/generate-helpers/pg-table-bindings/testgraphtables/testshortcircuit	[no test files]
FAIL
make: *** [Makefile:464: go-unit-tests] Error 2
INFO: Thu Aug 18 10:34:23 UTC 2022: Saving junit XML report
+ /go/bin/go-junit-report
go: downloading github.com/jstemmer/go-junit-report/v2 v2.0.0
/go/src/github.com/stackrox/stackrox/scripts/generate-junit-reports.sh
INFO: Thu Aug 18 10:34:25 UTC 2022: Copying test results from junit-reports to reports
Unit tests failed
INFO: Thu Aug 18 10:34:25 UTC 2022: Executing a general purpose exit trap for CI
Exit code is: 1
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-08-18T10:34:26Z"}
error: failed to execute wrapped command: exit status 1
---
Link to step on registry info site: https://steps.ci.openshift.org/reference/stackrox-initial
Link to job on registry info site: https://steps.ci.openshift.org/job?org=stackrox&repo=stackrox&branch=master&test=go-unit-tests 
INFO[2022-08-18T10:34:36Z] Reporting job state 'failed' with reason 'executing_graph:step_failed:executing_multi_stage_test'
</pre>
</details>

https://regex101.com/r/z0Husu/1
https://regex101.com/r/szqBNc/1

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -73,8 +73,8 @@ deck:
           - 'ERROR:'
           - ERRO\[
           - '^error:'
-          - (FAIL|Failure \[)\b
-          - panic\b
+          - '(^FAIL|FAIL: |Failure \[)\b'
+          - 'panic(.go)?:'
```
Comment (porridge) [inline core-services/prow/02_config/_config.yaml:77]:
```suggestion
          - 'panic(\.go)?:'
```

**Review by porridge — APPROVED** [review]
/lgtm
with a nitpick

**Comment by hongkailiu** [comment]
Sounds reasonable to me.


/approve

**Pull request merged**